pyproject.toml

```diff
@@ -237,14 +263,14 @@ max_supported_python = "3.13"
 [tool.pytest.ini_options]
 minversion = "8.0"
 testpaths = [ "tests" ]
+asyncio_mode = "auto"
+asyncio_default_fixture_loop_scope = "session"
 
 [tool.coverage.run]
-source_pkgs = [ "tika_client", "tests" ]
```

Comment:
This is how Hatch creates a project, so I don't think it needs to change.  I assume it is fine because hatch installs the project before testing

Reply:
this is not the hatch config, its the coverage config. You do not need to gather coverage from tests - actually you need to ignore them.

Reply:
When Hatch creates an project `hatch new myproject`, this is what it generates in the initial structure.  I would prefer to leave it as generated

Reply:
done